src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -729,6 +721,10 @@ std::string postUpdateContext
       // If some CER (but not all) fails, then it is a partial update
       parseDataP->upcrs.res.oe.fill(SccContextElementNotFound, "Attributes that were not updated: { " + failing + " }", "PartialUpdate");
     }
+    else {  // fails == 0
+      // No fail in this case, so invalidate any posible OrionError filled by mongoBackend on the mongoUpdateContext ste
```

Comment:
No failure, so invalidate any possible OrionError ...

Reply:
Fixed in 5a3ae39